dev/bots/test.dart

```diff
@@ -1180,14 +1180,30 @@ Future<void> _runWebLongRunningTests() async {
         driver: path.join('test_driver', 'integration_test.dart'),
         buildMode: buildMode,
         renderer: 'canvaskit',
+        expectWriteResponseFile: true,
+        expectResponseFileContent: 'null',
       ),
       () => _runFlutterDriverWebTest(
         testAppDirectory: path.join('packages', 'integration_test', 'example'),
         target: path.join('integration_test', 'extended_test.dart'),
         driver: path.join('test_driver', 'extended_integration_test.dart'),
         buildMode: buildMode,
         renderer: 'canvaskit',
-      ),
+        expectWriteResponseFile: true,
+        expectResponseFileContent: '''
+{
+  "screenshots": [
+    {
+      "screenshotName": "platform_name",
+      "bytes": []
+    },
+    {
+      "screenshotName": "platform_name_2",
+      "bytes": []
+    }
+  ]
+}''',
+    ),
```

Comment:
The indentation here is off:

```suggestion
      ),
```